### DISCUSSION TITLE
API for Shared Link and Embed Dashboard

### DISCUSSION THREAD

**Discussion by evanlegamer** [plausible/analytics#1030] [Anything Else]
Hi guys,

Is it currently possible to create share link and Embed dashboard by API ?

I would like to automatically show stats in my dashboard using the Embed Dashboard for websites, but currently we have to copy paste for each website and it's not possible when we scale and when we have thousand of customers.

Maybe it could be very convenient to propose this by API :)

Thank you in advance for your answer

**Comment by metmarkosaric** [comment]
hi! yes, that's possible. you can read more in our docs. see https://plausible.io/docs/sites-api

**Reply by evanlegamer** [reply]
oh, I hadn't seen that, I thought that when we transferred a shared link to an embed dashboard there was a manipulation behind it. Thank you so much ! Great job!
